src/DeterministicVI.jl

```diff
@@ -16,16 +16,23 @@ export ElboArgs
 
 """
 ElboArgs stores the arguments needed to evaluate the variational objective
-function
+function.
 """
 type ElboArgs{NumType <: Number}
     S::Int64
     N::Int64
+
+    # The number of components in the point spread function.
+    psf_K::Int64
     images::Vector{TiledImage}
     vp::VariationalParams{NumType}
     tile_source_map::Vector{Matrix{Vector{Int}}}
     patches::Matrix{SkyPatch}
     active_sources::Vector{Int}
+
+    # The number of standard deviations away below which a bivariate normal
+    # will be calculated.  Set to Inf to evaluate bivariate normals at all points.
+    num_allowed_sd::Float64
```

Comment:
These are all interesting ideas.

But as-is, this is a non-breaking, unit-tested experimental feature that leaves default functionality the same.  It is a potential fix to a potential problem that I just want to investigate as part of a bigger (and more pressing) problem.  I am not sure why you're pushing back so hard.  I think the potential downsides are clear as are the potential upsides and it is worth investigating.

Could I please just try it out?  Or should I just be forking Celeste so I can try stuff like this?